src/components/Checkbox.js

```diff
@@ -49,15 +49,22 @@ const Label = styled.span`
   margin-left: 0.5rem;
 `
 
-const Checkbox = ({ callback, checked, children, className, size = 2 }) => {
+const Checkbox = ({
+  callback,
+  checked,
+  children,
+  ariaContent,
+  className,
+  size = 2,
+}) => {
   const handleClick = () => {
     if (callback) {
       callback()
     }
   }
   return (
     <CheckboxContainer className={className} onClick={handleClick}>
-      <HiddenCheckbox checked={checked} readOnly />
+      <HiddenCheckbox checked={checked} readOnly aria-label={ariaContent} />
```

Comment:
Lets not define a new prop name (`ariaContent`) for this. We can do the following:
```jsx
const Checkbox = ({ callback, ...rest }) => {
  ....
  return (
    ...
    <HiddenCheckbox checked={checked} readOnly {...rest} />
  )
```

and from the consumer view, you can just use the native aria props
```jsx
<StyledCheckbox checked={isSelected} aria-label={title} />
```

Reply:
I made the changes, thanks for helping!